results.php

```diff
@@ -778,9 +778,47 @@ function assessOverallVals($var) {
 ?>
     </tbody>
     </table>
+<br>
```

Comment:
Real stuff starts here, the rest was done by atom automatically, removed whitespaces and stuff not needed.